pessimistic-transaction.md

```diff
@@ -139,6 +139,20 @@ TiDB supports the following two isolation levels in the pessimistic transaction
 
 - [Read Committed](/transaction-isolation-levels.md#read-committed-isolation-level). You can set this isolation level using the [`SET TRANSACTION`](/sql-statements/sql-statement-set-transaction.md) statement.
 
+## Pessimistic transaction commit process
+
+The commit phase of the pessimistic transaction mode and the optimistic transaction mode in TiDB has the same logic, and both commits are in the 2PC mode. The important adaptation of pessimistic transactions is DML execution.
+
+![TiDB pessimistic transaction commit logic](/media/pessimistic-transaction-commit.png)
+
+The pessimistic transaction adds an `Acquire Pessimistic Lock` phase before 2PC. This phase includes the following steps:
+
+1. (Same as the optimistic transaction mode) Receive the `begin` request from the client, and the current timestamp is this transaction's start_ts.
+2. When the TiDB server receives an `update` request from the client, the TiDB server initiates a pessimistic lock request to the TiKV server, and the lock is persisted to the TiKV server.
```

Comment:
We should be talking about "writing new data", rather than talking about `update` statements specifically.

Reply:
@MyonKeminta Please check if the following update is ok. Thanks.

![image](https://user-images.githubusercontent.com/79440533/191898287-46476287-b4f4-453f-b396-1ab1fbcc8d39.png)